deploy/operator.yaml

```diff
@@ -17,8 +17,8 @@ spec:
     spec:
       serviceAccountName: monstorak
       containers:
-      - image: uchapaga/monstorak:39d27f2
-        imagePullPolicy: Always
+      - image: uchapaga/monstorak:2f30eb8
```

Comment:
From private?

Reply:
Yup. We do not have any Quay/Docker organization.

Reply:
Can we use the same organization which Rook uses by any chance?

Reply:
We shouldn't.